files/generate_ansible_command.py

```diff
@@ -230,30 +230,35 @@ def get_hostname(x):
             # sure there is no funky chars for shell, and no space
             if re.search('^[\\w.-]+$', hostname):
                 # avoid using the ssh stuff on salt bus host
```

Comment:
is this comment thus not obsolete?

Reply:
yeah, I will clean later again, as I think there is a few old stuff. I just did that during a meeting :)